service/share/testing.go

```diff
@@ -242,3 +242,16 @@ func NewBrokenAvailability() Availability {
 func (b *brokenAvailability) SharesAvailable(context.Context, *Root) error {
 	return ErrNotAvailable
 }
+
+type TestAvailability struct {
```

Comment:
We already have `brokenAvailability` that always fails on `SharesAvailable`. May be we should give better naming to this one to be consistent ?

Reply:
We seem to be aligned with Dummy wording